machiavelli-cards/src/main/resources/schema.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA IF NOT EXISTS CARD;
```

Comment:
no need for separate schema per module

Reply:
jep, for standard things we can use a general schema - maybe game engine will use a separate one but no need to create separate schemas for such small collections